index.js

```diff
@@ -82,6 +92,18 @@ State.next = function() {
     web3.utils.toBN(State.block.nonce)
   );
 
+  let nextEpoch = ethHashUtil.getEpoc(nextBlockNumber);
+  if (State.epoch !== nextEpoch) {
+    console.log(`Block number: ${nextBlockNumber}`)
```

Comment:
Should be logged only if `LOGGING != "minimal"` ?

Reply:
I would leave it "as is" as it logs only when the epoch changes.